tests/test_managers/test_inheritance_manager.py

```diff
@@ -116,6 +116,17 @@ def test_prior_select_related(self):
                 "inheritancemanagertestchild2").get(pk=self.child1.pk)
             obj.inheritancemanagertestchild1
 
+    def test_select_subclass_num_queries(self):
+        with self.assertNumQueries(1):
+            objs = self.get_manager().select_subclasses(
+                "inheritancemanagertestchild1",
+                "inheritancemanagertestchild2",
+            )
+            list(objs)
+
+    def test_version_determining_any_depth(self):
+        self.assertIsNone(self.get_manager().all()._get_maximum_depth())
```

Comment:
It seems that `_get_maximum_depth()` was removed with https://github.com/jazzband/django-model-utils/commit/0fc0b44c95de3e39b71a1e01a3ef36a6bc887d12#diff-a4a84bac70b6f91df5344a194dbe270aL233